### ISSUE TITLE
[Python SDK] Unix domain sockets are not closed

### ISSUE THREAD

**Issue by georgebu** [Azure-Samples/cognitive-services-speech-sdk#2121]
**Describe the bug**

Unix domain sockets are not closed, causing file descriptors to leak.

**To Reproduce**

1. Put the code in _quickstart/python/text-to-speech/quickstart.py_ into a loop.
2. Each time `speech_synthesizer.speak_text_async(text).get()` is executed, two unix domain sockets are opened.
3. After many calls to the loop, execute `lsof -p {PID} | grep unix` on the terminal and you'll see a lot of unix domain sockets that are not closed. Like this:
```
python  PID USER   30u     unix 0x0000000000000000       0t0  5255951 type=STREAM
python  PID USER   31u     unix 0x0000000000000000       0t0  4891978 type=STREAM
python  PID USER   32u     unix 0x0000000000000000       0t0  4891979 type=STREAM
python  PID USER   33u     unix 0x0000000000000000       0t0  5255952 type=STREAM
...
```

**Expected behavior**

Unix domain sockets should be closed after speech synthesis.

**Version of the Cognitive Services Speech SDK**

Name: azure-cognitiveservices-speech
Version: 1.32.1
Summary: Microsoft Cognitive Services Speech SDK for Python

**Platform, Operating System, and Programming Language**

 - OS: Ubuntu 22.04.1
 - Hardware - x86_64
 - Programming language: Python

**Comment by pankopon** [comment]
There is a fix by @yulin-li that will be included in the Speech SDK 1.34.0 release before the end of this year.

**Comment by pankopon** [comment]
Closed as a fix has been released, please upgrade to the latest Speech SDK version and create a new issue if more support is needed.
